google-cloud-spanner/acceptance/spanner/fine_grain_access_control_test.rb

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "spanner_helper"
+
+describe "Fine Grained Access Control", :spanner do
+  let(:table_name) { "stuffs" }
+  let(:db) { spanner }
+  let(:db_client) { spanner_client }
+  let(:admin) { $spanner_db_admin }
+  let(:instance_id) { $spanner_instance_id }
+  let(:database_id) { $spanner_database_id }
+
+  before do
+    skip if emulator_enabled?
+    db_client.delete table_name # remove all data
+    db_client.insert table_name, [
+      { id: 1, bool: false },
+      { id: 2, bool: false },
+      { id: 3, bool: true },
+      { id: 4, bool: false },
+      { id: 5, bool: true }
+    ]
+
+    db_path = admin.database_path project: db.project_id,
+                                  instance: instance_id,
+                                  database: database_id
+
+    db_job = admin.update_database_ddl database: db_path, statements: [
+      "CREATE ROLE selector",
```

Comment:
I think it would be a good idea to create a "let" for the role name since it's used in multiple places throughout the tests.